cypress/e2e/find/redirect.feature

```diff
@@ -0,0 +1,17 @@
+Feature:
+
+  Redirect from old /find?serviceType={serviceType} to /find/{serviceType}
+
+  Scenario Outline: /find?serviceType=<serviceType> redirects to correct start page
+    When I navigate to "/find?serviceType=<serviceType><country>"
+    Then I should see the heading "<heading>"
+
+    Examples:
+      | serviceType             | country        | heading                                   |
+      | lawyers                 |                | Find a lawyer abroad                      |
+      | lawyers                 | &country=Italy | Find a lawyer in Italy                    |
+      | translatorsInterpreters |                | Find a translator or interpreter abroad   |
+      | translatorsInterpreters | &country=Italy | Find a translator or interpreter in Italy |
+      | funeralDirectors        |                | Find a funeral director abroad            |
+      | funeralDirectors        | &country=Italy | Find a funeral director in Italy          |
+
```

Comment:
Quick question about this test. I would have thought the other test files like `funeral-directors/answers.feature` and so on would kind of cover this 🤔

As in for that flow to work properly the urls need to work. Am I missing something?

Reply:
currently the URLs are `/find?serviceType=` - but this change has made it `/find/:serviceType`. I believe all the URLs on GOV.UK are /find?serviceType=.., so I need to ensure this URL isn't broken and redirects them to the right place. This suite tests 
1. the redirect works
2. the country parameter is retained and renders the right title